apiserver/common/networkingcommon/networkconfigapi_test.go

```diff
@@ -247,7 +247,7 @@ func (s *networkConfigSuite) TestUpdateMachineLinkLayerOpMultipleAddressSuccess(
 	s.expectMachine()
 	mExp := s.machine.EXPECT()
 	mExp.AllLinkLayerDevices().Return([]networkingcommon.LinkLayerDevice{lbDev, ethDev, delDev}, nil)
-	mExp.AllAddresses().Return([]networkingcommon.LinkLayerAddress{lbAddr, delAddr}, nil)
```

Comment:
This is the result of regenerating mocks. This change was missed back when the method was renamed.